filecoin-virtual-machine/section-a.md

```diff
@@ -0,0 +1,307 @@
+# Section A: Upload, PoDSI and Deal ID
+
+## Quickstart: Uploading Your First 🐕 Onto Filecoin
+
+### Upload your first file through the lighthouse SDK
+
+Firstly, you'll need to get a picture of your favorite pupper whose picture you'll want to immortalize on the internet.
+
+![Adapted from https://www.tldraw.com/r/v2_c_M7QpjoG42dpa5c2E4N2hG](../.gitbook/assets/DogDiagram.png)
+
+Think of the lighthouse SDK as a doggy daycare and the RaaS (renew, repair, replication) services as its caretaker. The lighthouse SDK provides a place for your puppy to stay while the services take care of your dog and makes sure it's fed and healthy. In this case, lighthouse provides storage while the RaaS services takes care of your file and makes sure it's stored on the Filecoin network permanently. 
+
+The lighthouse SDK is a JavaScript library that allows you to upload files to the Filecoin network. It's open source and available [here](https://github.com/lighthouse-web3/lighthouse-package)
+
+Uploading a file is as simple as:
+
+```javascript
+import lighthouse from "@lighthouse-web3/sdk";
+// ... other code
+const uploadResponse = await lighthouse.upload('/path/to/adorable/dog.jpg', 'YOUR_API_KEY');
+```
+
+Previously, if the file of your puppy was too small, it would encounter issues being stored on chain due to size minimums enforced by on-chain deal makers. The SDK helps you get around this by adding mock data to your file to meet the minimum size requirements.
+
+When you upload a file without any RaaS services, you've only uploaded one copy of your file to the Filecoin network. There's no guarantee that if the file's deal term expires or is no longer being maintained by the storage provider, that you'll be able to retrieve your file. This is where the RaaS services come in.
+
+Replication is the process of making copies of your file and storing them on the Filecoin network. This ensures that if one storage provider goes down, you'll still be able to retrieve your file from another storage provider.
+
+Here's an example of how to upload a file with replication:
+
+```javascript
+import lighthouse from "@lighthouse-web3/sdk";
+// ... other code
+// Indicates that the deal to maintain your file will be replicated to a total of two copies on the network.
+const dealParams = {
+  num_copies: 2,
+};
+// The `false` indicates that we're uploading a single file.
+// Returns a CID (Content ID) for your file that you can use for PoDSI verification.
+const uploadResponse = await lighthouse.upload('/path/to/adorable/dog.jpg', 'YOUR_API_KEY', false, dealParams);
+```
```

Comment:
Incorporate the content from (Appendix: Deal Parameters) here and no longer make it an appendix.